packages/foundation/docs/themes.mdx

```diff
@@ -1,30 +1,232 @@
 ---
-name: Default theme
+name: Documentation
 menu: Theme
 ---
 
 import * as packageJson from '../package.json';
 import ThemeTokens from '../../../src/ThemeTokens';
 import { DefaultTheme } from '../src/themes';
 
-# Themes
+# Theme
 
-@uireact provides with a default theme that you can use:
+@uireact provides with a default theme in the foundation package which can be found here: [Default theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-theme.ts)
 
-## Light coloration
+For understading the concepts and theme creation you should continue reading this doc ⚡️
 
-[Light Theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-colors/light.ts)
+_10 mins read_
+
+## Set up ⚙️
+
+@uireact uses a theme file so components can be customizable for different clients / use cases. We believe that
+our structure provides with a lot of customization and flexibility without getting into too much freedom which causes
+UX design to be poor for the user experience.
```

Comment:
I would probably say what is that it actually provides in favor of `We believe that our structure provides` that way it is very clear what this approach is proposing